Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16.16.0
+
+WORKDIR /repo
+COPY . .
+RUN npm install
+RUN npm run build
```

Comment:
I have a feeling that this step may do too much. Shouldn't we only run `lerna run build`? If so then probably I would need to add a script in `package.json`. Even if I am correct, I would prefer to address it in a separate PR.

Reply:
See https://github.com/streetsidesoftware/cspell/pull/3304#issuecomment-1195686997
If this PR is moved to cspell-cli, then `npm run build` is not needed.

Reply:
I agree. I created https://github.com/streetsidesoftware/cspell-cli/pull/183